index.html

```diff
@@ -259,4 +259,4 @@ <h3 class="text-3">LET’S CONNECT</h3>
     </main>
```

Comment:
- [ ] The mobile version of the portfolio should be closed to the original design Check the [templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) Kindly let's follow the original vision, you have to use the same colors and `background-color` to your buttons defined in the design if it's pretty and works, it will be a wonderful website. 🎨 🖌️ 

Figma Design | Your Design
------------------ | -----------------
![18](https://user-images.githubusercontent.com/79058364/185890977-08347f2d-8630-41a8-a24d-89f80853ce00.png) | ![18bad](https://user-images.githubusercontent.com/79058364/185891067-6b55163b-6aa5-433b-90a4-708ada628de7.png)




<img width="602" alt="Screen Shot 2022-08-19 at 10 22 09 PM" src="https://user-images.githubusercontent.com/83717285/185727145-0e9f990b-d2ca-4d2d-99a4-2b88a4f36f63.png">